rules/windows/persistence_startup_folder_scripts.toml

```diff
@@ -2,6 +2,8 @@
 creation_date = "2020/11/18"
 integration = ["endpoint", "windows"]
 maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
 updated_date = "2024/05/21"
```

Comment:
Bump update date

Reply:
Unit test identifying the same 😄  https://github.com/elastic/detection-rules/actions/runs/9810077631/job/27089419906?pr=3479 🚀